frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,153 @@
+import { SceneExport } from 'scenes/sceneTypes'
+import { useActions, useValues } from 'kea'
+import { useEffect } from 'react'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { urls } from 'scenes/urls'
+import { LemonButton, Link } from '@posthog/lemon-ui'
+import { onboardingLogic } from './onboardingLogic'
+import { billingProductLogic } from 'scenes/billing/billingProductLogic'
+import { convertLargeNumberToWords } from 'scenes/billing/billing-utils'
+import { BillingProductV2Type } from '~/types'
+import { LemonCard } from 'lib/lemon-ui/LemonCard/LemonCard'
+import { ProductPricingModal } from 'scenes/billing/ProductPricingModal'
+import { IconCheckCircleOutline, IconOpenInNew } from 'lib/lemon-ui/icons'
+
+export const scene: SceneExport = {
+    component: Onboarding,
+    // logic: featureFlagsLogic,
+}
+
+const OnboardingProductIntro = ({ product }: { product: BillingProductV2Type }): JSX.Element => {
+    const { currentAndUpgradePlans, isPricingModalOpen } = useValues(billingProductLogic({ product }))
+    const { toggleIsPricingModalOpen } = useActions(billingProductLogic({ product }))
+    const upgradePlan = currentAndUpgradePlans?.upgradePlan
+
+    const pricingBenefits = [
+        'Only pay for what you use',
+        'Control spend with billing limits as low as $0/mo',
+        'Generous free volume every month, forever',
+    ]
+
+    const productWebsiteKey = product.type.replace('_', '-')
+    const communityUrl = 'https://posthog.com/questions/topic/' + productWebsiteKey
+    const tutorialsUrl = 'https://posthog.com/tutorials/categories/' + productWebsiteKey
+    const productPageUrl = 'https://posthog.com/' + productWebsiteKey
+    const productImageUrl = `https://posthog.com/images/product/billing-products/${productWebsiteKey}.png`
```

Comment:
This is somewhat hacky because it's not concrete, but it's way easier than putting all these URLs in the billing service themselves. And if the urls on posthog.com change then we only have to change them once here rather than 5 places in the billing service. However it does require naming consistency between the billing service products and the posthog.com products, which could become an issue at some point in the future...